src/main/java/Command.java

```diff
@@ -0,0 +1,36 @@
+import java.io.IOException;
+
+/**
+ * The abstract Command class is used for extension to the different commands for different interaction
+ * with the saved TaskList and output printed to the user.
+ */
+abstract class Command {
```

Comment:
I like how you declared command class as abstract as different command types have different execute methods!